src/components/VBackToSearchResultsLink.vue

```diff
@@ -2,7 +2,7 @@
   <!-- @todo: Separate the absolute container from the link itself. -->
   <VLink
     v-if="show"
-    class="px-2 pt-1 md:px-6 md:pt-4 md:pb-2 flex flex-row items-center font-semibold text-dark-charcoal text-xs md:text-sr"
+    class="px-2 pt-3 pb-1 md:px-6 md:pt-4 md:pb-2 flex flex-row items-center font-semibold text-dark-charcoal text-xs md:text-sr"
```

Comment:
Could we use `py-X` instead of separate `pt` and `pb` values? Otherwise the spacing between the top and bottom of the link isn't symmetrical. While it's also not symmetrical on desktop, the tight margins of the mobile view combined with the proximity to the result (which isn't always the case on a wider viewport) make it look extra awkward. (I tried to upload a screenshot but I'm on a train and the Wifi isn't strong enough to support uploading at the moment).

Reply:
I checked the mockup, and we should use `py-3` class here.